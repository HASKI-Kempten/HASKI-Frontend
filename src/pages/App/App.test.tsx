import { App } from "@pages";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

describe("Test the demo component", () => {
  test("renders HASKI basic label", () => {
    const landingPage = render(<App />);
    expect(landingPage.getByText("HASKI")).toBeInTheDocument();
  });
});
