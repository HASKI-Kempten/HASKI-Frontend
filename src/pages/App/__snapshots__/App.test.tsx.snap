// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the demo component renders correctly 1`] = `
<div>
  HASKI
</div>
`;
